src/main/java/wellnus/common/MainManager.java

```diff
@@ -0,0 +1,157 @@
+package wellnus.common;
+
+import wellnus.command.CommandParser;
+import wellnus.exception.BadCommandException;
+import wellnus.manager.Manager;
+import wellnus.ui.TextUi;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
+
+public class MainManager extends Manager {
+    private static final String GREETING_MESSAGE = "Enter a command to start using WellNUS++! Try 'help' "
+            + "if you're new, or just unsure.";
+    private static final String INVALID_COMMAND_MESSAGE = "Don't recognise that command?";
+    private static final String INVALID_COMMAND_ADDITIONAL_MESSAGE = "Try 'help' for some guidance";
+    private static final String WELLNUS_FEATURE_NAME = "";
+    private final TextUi textUi;
+
+    public MainManager() {
+        this.textUi = new TextUi();
+    }
+
+    private static String getBriefAppDescription() {
+        return "WellNUS++ helps you keep track and improve your physical and mental wellness.";
+    }
+
+    private static String getLongAppDescription() {
+        return "WellNUS++ is a Command Line Interface(CLI) app for NUS Computing students to "
+                + "keep track and improve their physical and mental wellness in various aspects."
+                + " If you can type fast, WellNUS++ can update their wellness progress faster than "
+                + "traditional Graphical User Interface(GUI) apps.";
+    }
+
+    private void executeCommands() {
```

Comment:
You might want to add comments for this method to briefly introduce what it is doing since it is a main functionality for this class.